### PR TITLE
add cleanlab to Labeling Tools section

### PR DESCRIPTION
**Pull request by jwmueller**
cleanlab nicely complements any labeling tool and is very useful to assess data & label quality

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -333,6 +333,7 @@ This repository contains a curated list of awesome open source libraries that wi
 * [Semantic Segmentation Editor](https://github.com/Hitachi-Automotive-And-Industry-Lab/semantic-segmentation-editor) ![](https://img.shields.io/github/stars/Hitachi-Automotive-And-Industry-Lab/semantic-segmentation-editor.svg?style=social) - Hitachi's Open source tool for labelling camera and LIDAR data.
 * [Superintendent](https://github.com/janfreyberg/superintendent) ![](https://img.shields.io/github/stars/janfreyberg/superintendent.svg?style=social) - superintendent provides an ipywidget-based interactive labelling tool for your data.
 * [VGG Image Annotator (VIA)](http://www.robots.ox.ac.uk/~vgg/software/via/) - A simple and standalone manual annotation software for image, audio and video. VIA runs in a web browser and does not require any installation or setup.
+* [cleanlab](https://github.com/cleanlab/cleanlab) ![](https://img.shields.io/github/stars/cleanlab/cleanlab?style=social) - Python library for data-centric AI. Can automatically: find mislabeled data, detect outliers, estimate consensus + annotator-quality for multi-annotator datasets, suggest which data is best to (re)label next.
```
Comment (axsaucedo) [inline README.md:336]: Needs to go in alphabetical order

**Review by axsaucedo — CHANGES_REQUESTED** [review]
Thank you for the contribution, only change is it has to go in alphabetical order

File: README.md (hunk 1 above)
Comment (jwmueller) [reply README.md]: done!

**Comment by jwmueller** [comment]
moved the new section to respect alphabetical order

**Review by axsaucedo — APPROVED** [review]
Thanks - lgtm

**Pull request merged**